css/main.css

```diff
@@ -0,0 +1,203 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+.title1, .title2 {
+    display: flex;
+    float: 1 0;
+
+    justify-content: center;
+}
+.title1{
+    padding: 30px;
+    color: rgb(12, 79, 223);
+    border: 2px solid red;
+    margin: 10px auto;
+    width: 50%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.title2{
+    padding: 30px;
+    color: rgb(12, 79, 223);
+    border: 2px solid red;
+    margin: 0px auto;
+    width: 40%;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+}
+.title2 span{
+    color: white;
+    margin-left: 5px;
+}
+.header {
+    width: 100%;
+    background-image: url('../image/bg3.jpg');
+    background-position: 5px -100px;
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 50vh;
+    /* background: red; */
+}
+
+nav {
+    padding: 10px;
+    background-color: #333;
+    position: sticky;
+    height: 30px;
+    top: 0;
+    margin-bottom: 20px;
+}
+
+.item a {
+    text-decoration: none;
+    display: inline-block;
+    width: 150px;
+    padding: 10px;
+    color: rgb(240, 233, 233);
+    font-weight: 600;
+    text-align: center;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif
+}
+
+ul {
+    list-style-type: none;
+}
+
+.submenu-1 {
+    display: none;
+    position: absolute;
+    z-index: 2;
+    background-color: #333;
+    /* width: 300px; */
+}
+
+.submenu-1 li a {
+    width: 200px;
+    display: inline-block;
+    text-align: left;
+}
+
+li:not(.item):hover {
+    background-color: rgb(94, 94, 182);
+}
+
+.item:hover .submenu-1 {
+    display: block;
+}
+
+.item {
+    float: left;
+    border-right: rgb(240, 231, 231) 2px solid;
+}
+
+.container {
+    /* width: 100%; */
+    height: 60vh;
+    margin: 0 auto;
+    padding: 50px;
+}
+
+.btn {
+    display: block;
+    border: none;
+    padding: 10px;
+    width: 210px;
+    background: red;
+    opacity: 0.8;
+    font-size: 18px;
+    margin: 20px auto 10px auto;
+    color: white;
+    font-weight: 600;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.btn:hover {
+    background-color: green;
+    opacity: 0.8;
+}
+
+footer {
+    height: 20vh;
+    background-color: #333;
+    padding: 10px;
+}
+
+footer a {
+    display: block;
+    padding: 10px;
+    color: white;
+    text-align: left;
+    text-decoration: none;
+    font-weight: 600;
+}
+
+footer a:hover {
+    background: #ccc;
+    color: #333;
+}
+
+.footer-contianer {
+    float: left;
+    width: 33%;
+}
+
+/**for reg.html*/
+
+form {
+    width: 50%;
+    margin: 0 auto 10px auto;
+    border: 2px solid #ccc;
+    padding: 20px;
+}
```

Comment:
_There is no html tag avaliable in your html document_ please try to remove all css rules that have no any target tag in your document.